tests/lib/thrift/ThriftTest/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
-# Copyright 2022 Meta
-# SPDX-License-Identifier: Apache-2.0
+#Copyright 2022 Meta
+#SPDX - License - Identifier : Apache - 2.0
```

Comment:
?

Reply:
I had a pre-commit hook that was running `clang-format`, and it seemed to be formatting `CMakeLists.txt` too